docker/README.md

```diff
@@ -7,11 +7,32 @@ image (with CUDA support). The image comes with a pre-installed `sparseml`, as w
 
 This `Dockerfile` is tested on the Ubuntu 20.04.2 LTS with CUDA Version: 11.4.
 
+## Pull
+You can access the already built image detailed at https://github.com/orgs/neuralmagic/packages/container/package/sparseml:
+
+```bash
+docker pull ghcr.io/neuralmagic/sparseml:1.0.1-ubuntu18.04-cu11.1
+```
+
+## Extend
+If you would like to customize the docker image, you can use the pre-built images as a base in your own `Dockerfile`:
+
+```Dockerfile
+from ghcr.io/neuralmagic/sparseml:1.0.1-ubuntu18.04-cu11.1
```

Comment:
need to add example of either running the pulled image or tagging so it is compatible with the run example below

Reply:
@bfineran would including a `docker tag ghcr.io/... sparseml_docker` in the pull command be sufficient for this? Then the run commands in the run section could be used without updates I think

Reply:
Added the docker tag command.